lib/drive-get-experiment.js

```diff
@@ -0,0 +1,52 @@
+const Hyperdrive = require('hyperdrive')
+const Corestore = require('corestore')
+const Experiment = require('./experiment')
+
+const main = async () => { 
+  await drive.put('/blob.txt', Buffer.from('example'))
+
+  const buffer = await drive.get('/blob.txt')
+  console.log(buffer)
+
+  const entry = await drive.entry('/blob.txt')
+  console.log(entry)
+}
+
+class DriveGetExperiment extends Experiment {
+  constructor (tmpDir, { nrBlocks, blockByteSize }) {
+    super(tmpDir, 'Drive Get Hyperdrive experiment')
+
+    this.store = new Corestore(this.tmpDir)
+    this.drive = new Hyperdrive(this.store)
+    
+    this.nrBlocks = nrBlocks
```

Comment:
For hyperdrive, it's clearer to speak about `nrFiles` instead of `nrBlocks` (likewise in the lines below, name it something like: `fileByteSize` and `fileContent`)

Reply:
refactored all occurrences to `files`.